site/frontend/src/pages/compare/artifact-size/artifact-size-table.vue

```diff
@@ -0,0 +1,152 @@
+<script setup lang="ts">
+import {ArtifactDescription} from "../types";
+import {diffClass, formatSize} from "../shared";
+
+const props = defineProps<{
+  a: ArtifactDescription;
+  b: ArtifactDescription;
+}>();
+
+// Sort binaries first, libraries later. Then within each category, sort in descending order by size
+// of the original (a) artifact component.
```

Comment:
it's cute but the sorting feels a bit too clever: there's no visual separation between bins and libs (there could be an empty row) making the sections and their own sub-order hard to see and breaks scanning vertically; using one of the artifact sizes (which can evolve and result in different orderings between toolchains over time) can feel arbitrary as well as different from how we sort the other similar tables.

maybe it's fine, maybe "bins, empty row, libs -- each section alphabetically sorted" is clear enough: it's good to keep it simpler when presenting data.

Reply:
I added a new column to the table (Library/Binary) and changed ordering to be "binaries first, then alphabetically".